.github/workflows/verify-non-code.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: non-coding-checks
```

Comment:
`formatting-and-linting` or `code-integrity-checks` or `quality-check` or something like that?

Reply:
used `lint-and-spellcheck`